### DISCUSSION TITLE
Feedback on the tutorial on Booleans

### DISCUSSION THREAD

**Discussion by konnov** [informalsystems/quint#516] [Polls]
We are experimenting with different forms of tutorials for Quint. If you have finished the tutorial, please let us know, which kind of the tutorial you like better: the CodeTour tutorial, the Markdown tutorial, or both.

If you have any further suggestions about improving the tutorial, please leave a comment below.

**Comment by otrack** [comment]
at step #1, the file boolean.qnt is not shown in the editor
(ubuntu 20.04, vscode 1.77.3)

**Comment by JeftavanderHorst** [comment]
The tutorial is good, but I have several points of feedback:
- It was not clear to me if `and` and `all` are identical, same for `or` and `any`. As I understand, the only difference is that `and`/`or` typecheck for bools?
- The fact that `and { ... }` is syntax sugar for `and(...)` is surprising, does it really just swap the `(` for a `{` and add a space? What value does that add?
- The repeated blocks listing every possible call for each operator (like `quint "false and false"`) are a bit tedious
